webservices/tasks/download.py

```diff
@@ -140,6 +142,43 @@ def make_bundle(resource):
         )
 
 
+# modified for sqlalchemy 2.0 style taken from sqlalchemy-postgres-copy package by Joshua Carp
+# https://github.com/jmcarp/sqlalchemy-postgres-copy
+def copy_to(source, dest, engine, **flags):
+    dialect = postgresql.dialect()
+    statement = getattr(source, 'statement', source)
```

Comment:
I think maybe this isn't doing anything for us if we are using a select above right? Could we remove this line and change the next line to source, since we don't need to grab the select from the query object? 

compiled = source.compile(dialect=dialect)